### ISSUE TITLE
expo-doctor throws error due to notifee-expo-plugin

### ISSUE THREAD

**Issue by younes0** [evennit/notifee-expo-plugin#3]
I just updated expo-doctor and when I run it, it throws the following error:

```
root/.npm/_npx/c15e4b19a90d9797/node_modules/expo-doctor/build/index.js:89549
  throw new (_errors().PluginError)(`Failed to resolve plugin for module "${pluginReference}" relative to "${projectRoot}"`, 'PLUGIN_NOT_FOUND');
        ^

PluginError: Failed to resolve plugin for module "@evennit/notifee-expo-plugin" relative to "/projects/app/native"
    at resolvePluginForModule (/root/.npm/_npx/c15e4b19a90d9797/node_modules/expo-doctor/build/index.js:89549:9)
    at resolveConfigPluginFunctionWithInfo (/root/.npm/_npx/c15e4b19a90d9797/node_modules/expo-doctor/build/index.js:89598:7)
    at resolveConfigPluginFunction (/root/.npm/_npx/c15e4b19a90d9797/node_modules/expo-doctor/build/index.js:89589:7)
    at withStaticPlugin (/root/.npm/_npx/c15e4b19a90d9797/node_modules/expo-doctor/build/index.js:88669:70)
    at /root/.npm/_npx/c15e4b19a90d9797/node_modules/expo-doctor/build/index.js:88510:84
    at Array.reduce (<anonymous>)
    at withPlugins (/root/.npm/_npx/c15e4b19a90d9797/node_modules/expo-doctor/build/index.js:88510:18)
    at withConfigPlugins (/root/.npm/_npx/c15e4b19a90d9797/node_modules/expo-doctor/build/index.js:91158:47)
    at fillAndReturnConfig (/root/.npm/_npx/c15e4b19a90d9797/node_modules/expo-doctor/build/index.js:90080:78)
    at getConfig (/root/.npm/_npx/c15e4b19a90d9797/node_modules/expo-doctor/build/index.js:90133:12) {
  isPluginError: true,
  code: 'PLUGIN_NOT_FOUND',
  cause: undefined
}
```

**Comment by Kasendwa** [comment]
Hi @younes0, i've opened a PR to address this issue via #4. In the meantime you possibly should just apply this patch.

Path: `./patches/@evennit+notifee-expo-plugin+3.7.0.patch`

```
diff --git a/node_modules/@evennit/notifee-expo-plugin/app.plugin.js b/node_modules/@evennit/notifee-expo-plugin/app.plugin.js
new file mode 100644
index 0000000..eff60d2
--- /dev/null
+++ b/node_modules/@evennit/notifee-expo-plugin/app.plugin.js
@@ -0,0 +1 @@
+module.exports = require("./dist/plugin/withNotifee.js");
```

**Comment by kevpug** [comment]
I released the fix in v3.7.2 thanks to @Kasendwa!

**Comment by younes0** [comment]
thanks @Kasendwa @kevpug
